Terminal.Gui/Views/Button.cs

```diff
@@ -177,4 +177,12 @@ protected override void UpdateTextFormatterText ()
             }
         }
     }
+
+    /// <inheritdoc />
+    public bool LoadDemoData (object ctx = null)
```

Comment:
Obviously the signature would need to match the proposal for the interface modification.\
Here's a potentially nice way to do this default behavior:

```csharp
    public bool LoadDemoData<TContext> (in TContext buttonText) where TContext : notnull
    {
        Title = buttonText switch
                {
                    string => buttonText,
                    _=> "_Button"
                };
        return true;
    }
```

Note that the parameter name does not have to match for it to be an interface implementation. Nice way to give a little extra guidance to the consumer via expressive code.

Reply:
![image](https://github.com/gui-cs/Terminal.Gui/assets/585482/1747bb04-a6f9-49d7-964c-9833d481b0f1)

Reply:
Try with a cast `string => (string)buttonText,`.

Reply:
oh yeah. Forgot to say that the name of the type parameter is also not set in stone for implementations. Just have to match the same number of them and same constraints, but otherwise can call them whatever you like.